src/playbackstrategy/msestrategy.js

```diff
@@ -454,7 +454,15 @@ function MSEStrategy(mediaSources, windowType, mediaKind, playbackElement, isUHD
   }
 
   function getClampedTime(time, range) {
-    return Math.min(Math.max(time, range.start), range.end - Math.max(liveDelay, seekDurationPadding))
+    const vod = windowType === WindowTypes.STATIC
```

Comment:
can we change that to static? this will apply for audio/video

Reply:
I don't understand the question :face_with_spiral_eyes: I think this is independent of Audio/Video yes, but I don't do anything specific to audio/video?

Reply:
The const is called `vod` - but shouldn't really be names as something video specific, as it might be audio content.

Reply:
Ah yes, I can change that, I think that's just a factoring leftover